### DISCUSSION TITLE
Google Chrome -> Google Meet

### DISCUSSION THREAD

**Discussion by nicovoni** [ExistentialAudio/BlackHole#93] [General]
How should I use BlackHole to "share" the audio of a YouTube video opened in Google Chrome with the participants of a Google Meet meeting?
After installing BlackHole, in the Google Meet audio settings I selected BlackHole instead of microphone.  In this way I was no longer able to hear the voice of the other participants.  Furthermore, neither the participants nor myself can hear the audio from the YouTube video.  Most likely I have not configured the driver properly.  Can you explain how to do it on macOS Catalina?

**Comment by devinroth** [comment]
Please send some screenshots of how you have it set up.

**Comment by nicovoni** [comment]
> Please send some screenshots of how you have it set up.

I have not set anything.  I only installed BlackHole.  I want to clarify that Google Meet is also running in the Google Chrome browser.

**Comment by devinroth** [comment]
Try this first.

http://existential.audio/blackhole/support/#micsystem
